tests/test_admin_functions.py

```diff
@@ -0,0 +1,83 @@
+from admin import *
```

Comment:
[[Why is "import *" bad?](https://stackoverflow.com/questions/2386714/why-is-import-bad)](https://stackoverflow.com/questions/2386714/why-is-import-bad)

Read the article and correct it